torchinductor/lowering.py

```diff
@@ -2534,12 +2574,14 @@ def copy_(dst, src, non_blocking=False):
 add = register_pointwise(aten.add)
 div = register_pointwise(aten.div)
```

Comment:
this PR looks good now, but I think aten.div would still not promote both integer inputs, like eager does? Can we add it in this PR, or in a follow up?